components/footer/footer.tsx

```diff
@@ -55,20 +55,22 @@ export default function Footer() {
         >
           contact@finiam.com
         </Button>
-        <Button
-          className='cta-social'
-          buttonColor={BUTTON_COLORS.Transparent}
-          href={'https://twitter.com/wearefiniam'}
-        >
-          TW
-        </Button>
-        <Button
-          className='cta-social'
-          buttonColor={BUTTON_COLORS.Transparent}
-          href={'https://www.linkedin.com/company/finiam/'}
-        >
-          IN
-        </Button>
+        <div className={styles['wrapper-socials']}>
+          <Button
+            className='cta-social'
+            buttonColor={BUTTON_COLORS.Transparent}
+            href={'https://twitter.com/wearefiniam'}
```

Comment:
No big deal, but these can just be regular strings, like `href="https://twitter.com/wearefiniam"`